Sprints/sprint 8

BaronValeron opened the pull request.

BaronValeron committed:
Задание 8 спринта готово

BaronValeron committed:
Задание 8 спринта готово

DanilaG commented on ImageFeed/ImagesList/ImagesListViewController.swift:
```diff
@@ -0,0 +1,77 @@
+        
+        let imageInsets = UIEdgeInsets(top: 4, left: 16, bottom: 4, right: 16)
+        let cellWidth = tableView.bounds.width - imageInsets.left - imageInsets.right
+        let scale = image.size.height / image.size.width
```
Мало вероятно, но делитель может оказаться == 0. Тогда программа упадёт при попытке деления на 0. Давай добавим проверку деления на 0

DanilaG commented on ImageFeed/ImagesList/ImagesListCell.swift:
```diff
@@ -0,0 +1,18 @@
+final class ImagesListCell: UITableViewCell {
+    static let reuseIdentifier = "ImagesListCell"
+    
+    @IBOutlet var likeButton: UIButton!
+    @IBOutlet var dateLabel: UILabel!
+    @IBOutlet var cellImage: UIImageView!
```
Давай сделаем приватными, а ячейку будем конфигурировать через её метод. Тогда пользователи ячейки не будут зависеть от деталей её реализации

BaronValeron committed:
Устранено падение из-за date.now.
Визуальное отображение доработано и должно соответствовать макету
Учтены рекомендуемые исправления из комментариев к пулреквесту
